camayoc/tests/qpc/cli/test_sources.py

```diff
@@ -77,28 +79,68 @@ def generate_show_output(data):
         r"        }}\r\n"
         r"    \],\r\n".format(data["cred_name"])
     )
-    if data.get("exclude_hosts"):
-        output += r'    "exclude_hosts": \[\r\n' r'        "{}"\r\n' r"    \],\r\n".format(
-            data["exclude_hosts"]
-        )
-    output += r'    "hosts": \[\r\n' r'        "{}"\r\n' r"    \],\r\n".format(data["hosts"])
-    output += r'    "id": \d+,\r\n'
-    output += r'    "name": "{}",\r\n'.format(data["name"])
-    source_type = data["source_type"]
-    if source_type in QPC_HOST_MANAGER_TYPES:
-        data.setdefault("options", {}).setdefault("ssl_cert_verify", "true")
-    if data.get("options"):
-        output += '    "options": {\r\n'
-        output += ",\r\n".join(
-            [
-                '        "{}": ["]?{}["]?'.format(key, value)
-                for key, value in sorted(data["options"].items(), key=operator.itemgetter(0))
-            ]
+
+    # ─ disable_ssl ─
+    disable_ssl = data.get("disable_ssl")
+    output += r'    "disable_ssl": {},\r\n'.format(
+        str(disable_ssl).lower() if disable_ssl is not None else "null"
+    )
```

Comment:
Not a large problem, but I think `disable_ssl` follows the same logic as `use_paramiko`, so let's use the same pattern in both.

I think I like paramiko approach better, but it seems we can still avoid repetition of appending to output in both branches.